test/src/tests-forwarders.js

```diff
@@ -2738,6 +2738,209 @@ describe('forwarders', function() {
+
+                    mParticle.logEvent('Test Event2');
+
+                    // SideloadedKit11 receives Test Event2, but SideloadedKit22 does not
+                    window.SideloadedKit11.instance.receivedEvent.EventName.should.equal('Test Event2');
+                    window.SideloadedKit22.instance.receivedEvent.EventName.should.not.equal('Test Event2');
+                });
+
+                it('should filter event types out properly when set', function() {
+                    mpSideloadedKit1.addEventTypeFilter(mParticle.EventType.Unknown);
+                    mpSideloadedKit2.addEventTypeFilter(mParticle.EventType.Navigation);
+
+                    const sideloadedKits = [mpSideloadedKit1, mpSideloadedKit2];
+
+                    window.mParticle.config.sideloadedKits = sideloadedKits;
+
+                    mParticle.init(apiKey, window.mParticle.config);
+                    mParticle.logEvent('Test Event', mParticle.EventType.Unknown);
+
+                    // The received event gets replaced by the last event sent to the forwarder
+                    // SideloadedKit11 has received the session start event, but not the Test Event of EventType.Unknown 
+                    // SideloadedKit22 will receive the Test Event of EventType.Unknown 
+                    window.SideloadedKit11.instance.receivedEvent.EventName.should.not.equal('Test Event');
+                    window.SideloadedKit22.instance.receivedEvent.EventName.should.equal('Test Event');
+
+                    mParticle.logEvent('Test Event2', mParticle.EventType.Navigation);
+
+                    // SideloadedKit11 receives the Navigation Event, SideloadedKit22 does not
+                    window.SideloadedKit11.instance.receivedEvent.EventName.should.equal('Test Event2');
+                    window.SideloadedKit22.instance.receivedEvent.EventName.should.not.equal('Test Event2');
+                });
+
+                it('should filter event attributes out properly when set', function() {
+                    mpSideloadedKit1.addEventAttributeFilter(mParticle.EventType.Navigation, 'Test Event', 'testAttr1');
+                    mpSideloadedKit2.addEventAttributeFilter(mParticle.EventType.Navigation, 'Test Event', 'testAttr2');
+
+                    const sideloadedKits = [mpSideloadedKit1, mpSideloadedKit2];
+
+                    window.mParticle.config.sideloadedKits = sideloadedKits;
+
+                    mParticle.init(apiKey, window.mParticle.config);
+
+                    const attrs = {
+                        testAttr1: 'foo',
+                        testAttr2: 'bar'
+                    }
+
+                    mParticle.logEvent('Test Event', mParticle.EventType.Navigation, attrs);
+
+                    window.SideloadedKit11.instance.receivedEvent.EventAttributes.should.have.property('testAttr2', 'bar');
+                    window.SideloadedKit11.instance.receivedEvent.EventAttributes.should.not.property('testAttr1');
+s
```

Comment:
This stray `s` is breaking the build.

```suggestion
```

Reply:
Whoops I saw that after the fact and didn't commit it...